src/System.Runtime.InteropServices.RuntimeInformation/src/Configurations.props

```diff
@@ -2,17 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      win8-Windows_NT;
-      wpa81-Windows_NT;
-      net45-Windows_NT;
-      netfx-Windows_NT;
-      net462-Windows_NT;
-      net47-Windows_NT;
       uap-Windows_NT;
```

Comment:
Do we not need a new netfx configuration for this?

Reply:
No, we no longer build that in corefx now that its part of netstandard.  It got absorbed into the framework.

Reply:
ah yeah :)

Reply:
@ericstj will you fix the failure ``` Could not load file or assembly 'System.Collections.Immutable ``` with your changes in buildtools? or you want me or @safern take care of it?

Reply:
He took care of it on: https://github.com/dotnet/buildtools/pull/1649